app/models/user.rb

```diff
@@ -12,6 +13,36 @@ class User < ApplicationRecord
     length: {minimum: Settings.length.digit_6}, if: :password
   has_secure_password
 
+  class << self
+    def digest string
+      cost = if ActiveModel::SecurePassword.min_cost
+               BCrypt::Engine::MIN_COST
+             else
+               BCrypt::Engine.cost
+             end
+      BCrypt::Password.create string, cost: cost
+    end
+
+    def new_token
+      SecureRandom.urlsafe_base64
+    end
+  end
+
+  def remember
+    self.remember_token = User.new_token
+    update remember_digest: User.digest(remember_token)
```

Comment:
```suggestion
    update_column :remember_digest, User.digest(remember_token)
```
tốt nhất ở đây nên dùng update column vì chỗ này em ko cần validate hay call callback j cả nè